### ISSUE TITLE
Can Ice support Swoole?

### ISSUE THREAD

**Issue by ashokkumar88** [ice/framework#291]

**Comment by mruz** [comment]
No, but you can use https://github.com/ice/framework/tree/dev/ice/cli/websocket for the asynchronous connections.
Maybe we could add support at some point.
